src/expandableCalendar/week.tsx

```diff
@@ -31,7 +31,7 @@ const Week = (props: WeekProps) => {
   const renderDay = (day: XDate, id: number) => {
     const dayProps = extractDayProps(props);
     const currXdate = parseDate(current);
-    
```

Comment:
Please remove the changes here. There is no need to include this file if there are no actual changes here